README.md

```diff
@@ -228,7 +228,7 @@ The DSL is based on a subset of language recognized by [typify-parser](https://g
 - *functions* are supported: `"bool -> bool"` is evaluated to `jsc.fn(jsc.bool)`.
 - *square brackets* are treated as a shorthand for the array type: `"[nat]"` is evaluated to `jsc.array(jsc.nat)`.
 - *union*: `"bool | nat"` is evaluated to `jsc.sum([jsc.bool, jsc.nat])`.
-    - **Note** `oneof` cannot be shrinked, because the union is untagged, we don't know which shrink to use.
+    - **Note** `sum` cannot be shrinked, because the union is untagged, we don't know which shrink to use.
```

Comment:
The note is right when mentioning `oneof`. It's too terse, there isn't enougb context:

We use `sum` because it can be shrinked, and the result of `bool | nat` is **tagged** union. For untagged one could use `oneof`, but that cannot be properly shrinked.

---

Btw, the README.md is generated file. Please read https://github.com/jsverify/jsverify/blob/master/CONTRIBUTING.md